start.sh

```diff
@@ -0,0 +1,3 @@
+#!bin/bash
+
```

Comment:
Super nitpicky, but it may be good to ensure ```npm start``` is executed from the correct directory. Perhaps ```cd $(dirname $0)``` or something similar to make sure the script's working directory is always the directory it's located in, rather than the one it's launched from. Depending on the deployment, it may not be an issue but it's definitely something to consider.

Reply:
This isn't used in prod deployments for docker. This existed for testing locally but in prod we would use nginx to serve the files instead.